datastore/usermanager.go

```diff
@@ -217,6 +218,16 @@ func (db *DB) GetUserByUsername(username string) (User, error) {
 	return user, err
 }
 
+// GetUserByAPIKey fetches a single user from database
+func (db *DB) GetUserByAPIKey(apiKey, username string) (User, error) {
```

Comment:
I guess GetUserByAPIKeyAndUsername is a bit verbose, isn't it?

Reply:
Yes, that's what I was thinking